src/modules/fancyzones/lib/ZoneWindow.cpp

```diff
@@ -54,7 +61,7 @@ namespace ZoneWindowDrawUtils
     {
         Gdiplus::Graphics g(hdc.get());
 
-        Gdiplus::FontFamily fontFamily(L"Segoe ui");
+        Gdiplus::FontFamily fontFamily(NonLocalizable::ZoneIndexFont);

```

Comment:
Segoe ui is font name, so ZoneIndexFont isn't the best replacement IMHO
ZoneIndexFont -> SegoeUiFont (or just SegoeUi)

Reply:
... SegoeUiStr

Reply:
Done.